common/app/common/pages/helpers.cljc

```diff
@@ -410,3 +459,29 @@
         [parent-idx _] (d/seek (fn [[idx child-id]] (= child-id shape-id))
                                (d/enumerate (:shapes parent)))]
     parent-idx))
+
+(defn remove-children
```

Comment:
Looks like this function has the same purpose than `app.common.page.helpers/clean-loops`, i'm wrong?

Reply:
You're right, i'll change to that function.